Makefile

```diff
@@ -277,10 +277,14 @@ dist/$(PACKAGE_NAME)-$(PACKAGE_VERSION)-build-epoch.txt:
 	echo $(SOURCE_DATE_EPOCH) > dist/$(PACKAGE_NAME)-$(PACKAGE_VERSION)-build-epoch.txt
 
 # Build the HTML documentation from the package's source.
-DOCS_SOURCE := $(shell find docs/source -type f -name '*.rst')
+DOCS_SOURCE := $(shell find docs/source -type f -name '*.rst' -or -name '*.py' -or \
```

Comment:
Hmm, how about we use this instead:
```make
DOCS_SOURCE := $(shell git ls-files docs/source)
```
This makes doc sources all files under `docs/sources`, and we don't have to maintain a list of extensions in the `find` command, which can be pretty error-prone and hard to maintain. I think the current list is missing `*.html` for the sample HTML reports and `*.svg` for some diagrams, for example.

Reply:
@nathanwn Thanks for the suggestion. See commit [786e599](https://github.com/oracle/macaron/pull/370/commits/786e599bb509590c9311b06d4c76347ea6c541f9).